catroid/src/main/java/org/catrobat/catroid/io/asynctask/ProjectLoadTask.java

```diff
@@ -67,7 +67,11 @@ public static boolean task(File projectDir, Context context) {
 			new DeviceListAccessor(projectDir).cleanUpDeletedUserData(project);
 			return true;
 		} catch (Exception e) {
-			Log.e(TAG, "Cannot load project in " + projectDir.getAbsolutePath(), e);
+			if (projectDir != null) {
+				Log.e(TAG, "Cannot load project in " + projectDir.getAbsolutePath(), e);
+			} else {
+				Log.e(TAG, "Cannot load project due to null projectDir");
+			}
```

Comment:
could you please explain how this will fix the bug? 
as I can see you added a test and some logs inside the catch block, did you miss committing and pushing the code which might fix the bug?